src/components/FloatingFlow.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+  <el-container style="height:100%;background:white;">
+    <el-header style="text-align: left; font-size: 14px;padding:0" height="40px" class="dialog-header">
+      <DialogToolbarContent :activeId="activeDockedId" :dialogTitles="dockedArray"
+        :showIcons="entries[findIndexOfId(activeDockedId)].mode!=='main'" @maximise="dockedMaximise"
```

Comment:
So icons need to be hidden based on which dialog is the main one?

Reply:
Yes, the flatmap dialog can never be destroyed.

Reply:
Ok so icons here relate to display of the flatmap dialog?

Reply:
Yes, when mode is "main" means that it is the main dialog, should and cannot not be destroyed.